R/RLearner_regr_gstat.R

```diff
@@ -0,0 +1,164 @@
+      makeUntypedLearnerParam(id = "beta.gstat"),
+      makeIntegerLearnerParam(id = "nmax", default = 0),
+      makeIntegerLearnerParam(id = "nmin", default = 0),
+      makeIntegerLearnerParam(id = "omax", default = 0),
+      makeNumericLearnerParam(id = "maxdist", allow.inf = TRUE, default = Inf),
+      makeLogicalLearnerParam(id = "dummy", default = FALSE),
+      makeUntypedLearnerParam(id = "set"),
+      makeFunctionLearnerParam(id = "x"),
+      makeLogicalLearnerParam(id = "fill.all", default = FALSE),
+      makeLogicalLearnerParam(id = "fill.cross", default = TRUE),
+      makeDiscreteLearnerParam(id = "variance", values = c("identity", "mu", "mu(1-mu)"), default = "identity"),
+      makeUntypedLearnerParam(id = "merge"),
+      makeIntegerLearnerParam(id = "degree", default = 0),
+      makeLogicalLearnerParam(id = "vdist", default = FALSE),
+      # gstat::variogram params
+      makeNumericLearnerParam(id = "cutoff"), # FIXME default value
+      makeNumericLearnerParam(id = "width"),
+      makeNumericLearnerParam(id = "alpha", default = 0),
+      makeNumericLearnerParam(id = "beta.variogram", default = 0),
+      makeNumericLearnerParam(id = "tol.hor"),
+      makeNumericLearnerParam(id = "tol.ver"),
+      makeLogicalLearnerParam(id = "cressie", default = FALSE),
+      makeNumericLearnerParam(id = "dX", default = 0),
+      makeNumericLearnerParam(id = "boundaries", default = 0),
+      makeLogicalLearnerParam(id = "cloud", default = FALSE),
+      makeUntypedLearnerParam(id = "trend.beta", default = NULL),
+      makeIntegerLearnerParam(id = "debug.level", default = 1),
+      makeLogicalLearnerParam(id = "cross", default = TRUE),
+      makeLogicalLearnerParam(id = "map", default = FALSE),
+      makeLogicalLearnerParam(id = "projected", default = TRUE),
+      makeNumericLearnerParam(id = "lambda", default = 1.0),
+      makeLogicalLearnerParam(id = "verbose", default = FALSE),
+      makeLogicalLearnerParam(id = "covariogram", default = FALSE),
+      makeLogicalLearnerParam(id = "PR", default = FALSE),
+      makeIntegerLearnerParam(id = "pseudo", default = -1),
+      # gstat::fit.variogram params
+      makeLogicalLearnerParam(id = "fit.sills", default = TRUE),
+      makeLogicalLearnerParam(id = "fit.ranges", default = TRUE),
+      makeIntegerLearnerParam(id = "fit.method", default = 7),
+      makeLogicalLearnerParam(id = "warn.if.neg", default = FALSE),
+      makeLogicalLearnerParam(id = "fit.kappa", default = FALSE),
+      # gstat::vgm params
+      makeUntypedLearnerParam(id = "psill", default = NA),
+      makeUntypedLearnerParam(id = "model"), # FIXME what should be the type ?
+      makeUntypedLearnerParam(id = "range", default = NA),
+      makeNumericLearnerParam(id = "kappa", default = 0.5),
+      makeUntypedLearnerParam(id = "nugget"),
+      makeUntypedLearnerParam(id = "add.to"),
+      makeUntypedLearnerParam(id = "covtable"),
+      makeNumericLearnerParam(id = "Err", default = 0)
```

Comment:
Can you provide ranges for the parameters where possible please? Do e.g. all the numeric parameters from -inf to inf?

Reply:
Sure, I'll do this